etc/env/example.env

```diff
@@ -9,6 +9,7 @@ CANDIG_AUTH_MODULES=keycloak tyk opa vault
 CANDIG_DOMAIN=docker.localhost
 CANDIG_AUTH_DOMAIN=docker.localhost
 CANDIG_SITE_LOCATION=UHN
+CANDIG_DEBUG_MODE=1
```

Comment:
Is debug what we want by default?

Reply:
I think so...I'd imagine that prod is the only one that we wouldn't want on debug mode.